src/stan_math_backend/Cpp.ml

```diff
@@ -380,6 +381,7 @@ module Printing = struct
     | TemplateType id -> pp_identifier ppf id
     | StdVector t -> pf ppf "@[<2>std::vector<@,%a>@]" pp_type_ t
     | Array (t, i) -> pf ppf "@[<2>std::array<@,%a,@ %i>@]" pp_type_ t i
+    | Tuple ts -> pf ppf "@[<2>std::tuple<@,%a>@]" (list ~sep:comma pp_type_) ts
```

Comment:
Well, that's new.
It's not used anywhere. Could you add a `let%expect_test` for this?

Reply:
Oops, that's for #1100 (where I first spotted this) but shouldn't have snuck in here!